### ISSUE TITLE
Navigator Start Error

### ISSUE THREAD

**Issue by thermike2** [ContinuumIO/anaconda-issues#7003]
After installing Anaconda 5.0 I can't start Anaconda Navigator. I get the following error window:

Traceback (most recent call last):
File "C:\Users\Mike\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 636, in wrapper
return fun(self, *args, **kwargs)
File "C:\Users\Mike\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 706, in cmdline
ret = cext.proc_cmdline(self.pid)
PermissionError: [WinError 5] Access is denied

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "C:\Users\Mike\Anaconda3\lib\site-packages\anaconda_navigator\exceptions.py", line 75, in exception_handler
return_value = func(*args, **kwargs)
File "C:\Users\Mike\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 113, in start_app
if misc.load_pid() is None: # A stale lock might be around
File "C:\Users\Mike\Anaconda3\lib\site-packages\anaconda_navigator\utils\misc.py", line 384, in load_pid
cmds = process.cmdline()
File "C:\Users\Mike\Anaconda3\lib\site-packages\psutil\__init__.py", line 721, in cmdline
return self._proc.cmdline()
File "C:\Users\Mike\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 639, in wrapper
raise AccessDenied(self.pid, self._name)
psutil.AccessDenied: psutil.AccessDenied (pid=2528)

I tried conda update anaconda
conda update <packages> with no luck

Any suggestions? Thanks

**Comment by goanpeca** [comment]
**See issue #1984 for more information on how to fix this.**

---

Closing as duplicate of #1984

---

Please remember to update to the latest version of Navigator to include
the latest fixes.

Open a terminal (on Linux or Mac) or the Anaconda Command Prompt (on windows)
and type:

```
$ conda update anaconda-navigator
$ conda update navigator-updater
```

**Comment by mingwandroid** [comment]
@goanpeca, this is on Anaconda 5 so the bug isn't likely to do with old navigator is it?

**Comment by csoja** [comment]
I asked @goanpeca about these errors - and apparently it is related to how Navigator handles locking on Windows.  He did not yet have a fix included in Navigator as of 5.0 as far as I know.  If the users tries "anaconda-navigator --reset", it should resolve the problem.
